exporter.go

```diff
@@ -534,6 +658,20 @@ func (e *Exporter) addOrReuseLayer(image imgutil.Image, layer layers.Layer, prev
 	return layer.Digest, image.AddLayerWithDiffID(layer.TarPath, layer.Digest)
 }
 
+func (e *Exporter) addOrReuseExtensionLayer(image imgutil.Image, layer layers.Layer) (string, error) {
+	rc, err := image.GetLayer(layer.Digest)
+	if err != nil {
+		// assume we failed to get the layer because it doesn't exist
+		e.Logger.Infof("Adding layer %s\n", layer.ID)
```

Comment:
nit: I would prefer unique log lines for extension related output
```suggestion
		e.Logger.Infof("Adding extension layer %s\n", layer.ID)
```